boot/src/test/java/com/taobao/arthas/boot/DownloadUtilsTest.java

```diff
@@ -0,0 +1,38 @@
+package com.taobao.arthas.boot;
+
+import com.taobao.arthas.common.IOUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+
+public class DownloadUtilsTest {
+
+    @Test
+    public void testReadMavenReleaseVersion() {
+        Assert.assertNull(DownloadUtils.readMavenReleaseVersion(""));
+    }
+
+    @Test
+    public void testReadAllMavenVersion() {
+        Assert.assertEquals(new ArrayList<String>(), DownloadUtils.readAllMavenVersion(""));
+    }
+
+    @Test
+    public void testGetRepoUrl() {
+        Assert.assertEquals("http", DownloadUtils.getRepoUrl("https/", true));
+        Assert.assertEquals("https://repo1.maven.org/maven2", DownloadUtils.getRepoUrl("center", false));
+        Assert.assertEquals("https://maven.aliyun.com/repository/public", DownloadUtils.getRepoUrl("aliyun", false));
+    }
+
+    @Test
+    public void testReadMavenMetaData() throws IOException {
+        Assert.assertEquals(IOUtils.toString(new URL("http://repo1.maven.org/maven2/com/taobao/arthas/arthas-packaging/maven-metadata.xml").openStream()), DownloadUtils.readMavenMetaData("center", true));
+        Assert.assertEquals(IOUtils.toString(new URL("http://archiva-maven-storage-prod.oss-cn-beijing.aliyuncs.com/repository/central/com/taobao/arthas/arthas-packaging/maven-metadata-public.xml?Expires=1555349003&OSSAccessKeyId=LTAIfU51SusnnfCC&Signature=L6XWC%2BKA1wa0g%2BrpqvMP7EqrRzs%3D").openStream()), DownloadUtils.readMavenMetaData("aliyun", true));
```

Comment:
The assert on line 33 is ugly (and very long) whereas the assert on line 32 is less so. Can this be improved?